db/mongo.js

```diff
@@ -0,0 +1,26 @@
+const { MongoClient } = require('mongodb');
+
+const dbName = 'abstgtplregistryva6';
+const url = process.env.MONGODB_URI || 'mongodb://localhost:27017';
```

Comment:
This won't work when deployed to Runtime since action inputs aren't added to `process.env` automatically, they're passed to the action main via `main(params)`